ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt

```diff
@@ -56,10 +59,52 @@ class IdeaFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) : Co
 
     fun openProjectStructure() = step("Open Project Structure dialog") {
         if (remoteRobot.isMac()) {
-            keyboard { this.hotKey(KeyEvent.VK_META, KeyEvent.VK_SEMICOLON) }
+            keyboard { hotKey(KeyEvent.VK_META, KeyEvent.VK_SEMICOLON) }
         } else {
-            keyboard { this.hotKey(KeyEvent.VK_SHIFT, KeyEvent.VK_SHIFT, KeyEvent.VK_ALT, KeyEvent.VK_S) }
+            keyboard { hotKey(KeyEvent.VK_SHIFT, KeyEvent.VK_SHIFT, KeyEvent.VK_ALT, KeyEvent.VK_S) }
         }
         find(ComponentFixture::class.java, byXpath("//div[@accessiblename='Project Structure']")).click()
     }
+
+    // Show AWS Explorer, or leave it open if it is already open
+    fun showAwsExplorer() {
+        try {
+            find<AwsExplorer>(byXpath("//div[@class='ExplorerToolWindow']"))
+        } catch (e: Exception) {
+            find(ComponentFixture::class.java, byXpath("//div[@accessiblename='AWS Explorer' and @class='StripeButton' and @text='AWS Explorer']")).click()
+        }
+    }
+
+    fun setCredentials(profile: String, region: String) {
+        openCredentialsPanel()
+        // This will grab both the region and credentials
+        findAll<JListFixture>(byXpath("//div[@class='MyList']")).forEach {
+            if (it.items.contains(profile)) {
+                it.selectItem(profile)
+            }
+        }
+        openCredentialsPanel()
+        findAll<JListFixture>(byXpath("//div[@class='MyList']")).forEach {
+            if (it.items.contains(region)) {
+                it.selectItem(region)
+            }
+        }
+    }
+
+    // Tips sometimes open when running locally, close it if it opens
+    fun tryCloseTips() {
+        try {
+            find<ComponentFixture>(byXpath("//div[@accessiblename='Close' and @class='JButton' and @text='Close']")).click()
+        } catch (e: Exception) {
+        }
+    }
+
+    private fun openCredentialsPanel() = try {
+        // 2020.1
+        findAndClick("//div[@class='MultipleTextValues']")
```

Comment:
You should be able to match this by `accessiblename='AWS Connection Setting'`,

I believe I used that somewhere else in the code base

Reply:
The accessible name is actually what it is set to: `@accessiblename='AWS: Profile:default@US West (Oregon)` so we can't do that

Reply:
Really? UI Inspector showed different, even then XPath has `starts-with()`.
MultipleTextValues feels not specific enough IMO

Reply:
It's multipletextvalues in the top level frame though, which should be enough. I'd like to replace this when connection ux merges as well so I don't want to think about it too much.